docs/user/troubleshooting.rst

```diff
@@ -0,0 +1,286 @@
+- running BEM codes
+- use or development of BEM codes
+
+The issue board is provided as a convenience to users and the development team makes every effort to respond to issues each week. 
+Issues are not addressed over weekends nor U.S. holidays.
+Users should not expect nor request an immediate response from the developers.
+
+
+Numerical Test Cases
+--------------------
+This section describe a series of numerical test cases that should be performed when creating a novel WEC-Sim case.
+These various wave cases are necessary to ensure a robust, accurate solution and speed the debugging process for users.
+When opening a support question for case development, users will be asked to supply information on which test cases are functioning or not.
+Note that this workflow is not foolproof, but can be used as a guide to create a more robust solution when developing a WEC-Sim case.
+
+======  =================================  =========================
+Number  Purpose                            Input parameters utilized
+======  =================================  =========================
+1A      Hydrostatic stability              noWave
+1B      Hydrostatic stability              noWaveCIC
+2A      Decay test, hydrostatic stiffness  noWave, initDisp
+2B      Decay test, hydrostatic stiffness  noWaveCIC, initDisp
+3A      Viscous drag                       regular
+3B      Viscous drag                       regularCIC
+4A      Full functionality                 irregular, initDisp
+======  =================================  =========================
+
+Various problems may while progressing through these test cases.
+Users should not advance to the next test until the previous ones run without error and with physical responses.
+
+Test 1:
+^^^^^^^
+
+Issues with Test 1A-B indicate there is an imbalance between the gravity and buoyant forces. 
+This may cause the "solution singularity" as described in the FAQ, or result in a body rising or falling indefinitely.
+To solve this problem, recalculate the mass that will balance the displaced volume from the BEM data.
+Alteratively utilize the ``body(#).mass = equilibrium`` option.
+
+Test 2:
+^^^^^^^
+
+Failure in Test 2 but not Test 1 inidicates an inaccurate hydrostatic stiffness.
+The hydrostatic stiffness returns a device to equilibrium after some displacement.
+If the stiffness is too large, the simulation may require a very small time step. 
+If too small, an initial displacement may still cause infinite motion.
+Try altering the stiffness by using ``body(#).hydroStiffness`` in the input file.
+
+Test 3:
+^^^^^^^
+
+A hydrostatically stable device that has an unphysical response to a regular wave requires different drag and damping.
+Viscous drag is essential to a physical response in WEC-Sim.
+Tune the parameters ``body(#).viscDrag`` or ``body(#).linearDamping`` to cause a more realistic response.
+
+Test 4:
+^^^^^^^
+
```

Comment:
Any suggestions if it fails test 4?

Reply:
Good point, the tests get more difficult to debug as they get more complicated. I will put in a note for users to check the IRF in BEMIO as for the 'B' tests. Also users can try the state space, or changing stiffness, damping, etc. I'll make notes of this